components/arm/universalrobots/ur5e_test.go

```diff
@@ -123,42 +123,43 @@ func TestKin1(t *testing.T) {
 	))
 }
 
-func TestUseURHostedKinematics(t *testing.T) {
-	sphere, err := spatialmath.NewSphere(spatialmath.NewZeroPose(), 1, "sphere")
-	test.That(t, err, test.ShouldBeNil)
-	obstacles := []spatialmath.Geometry{sphere}
-	gifs := []*referenceframe.GeometriesInFrame{referenceframe.NewGeometriesInFrame(referenceframe.World, obstacles)}
-
-	// test that under normal circumstances we can use worldstate and our own kinematics
-	ur := URArm{}
-	using, err := ur.useURHostedKinematics(&referenceframe.WorldState{Obstacles: gifs}, nil)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, using, test.ShouldBeFalse)
-
-	// test that extra params can be used to get the arm to use the hosted kinematics
-	extraParams := make(map[string]interface{})
-	extraParams["arm_hosted_kinematics"] = true
-	using, err = ur.useURHostedKinematics(nil, extraParams)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, using, test.ShouldBeTrue)
-
-	// test specifying at config time with no obstacles or extra params at runtime
-	ur.urHostedKinematics = true
-	using, err = ur.useURHostedKinematics(&referenceframe.WorldState{}, nil)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, using, test.ShouldBeTrue)
-
-	// test that we can override the config preference with extra params
-	extraParams["arm_hosted_kinematics"] = false
-	using, err = ur.useURHostedKinematics(&referenceframe.WorldState{Obstacles: gifs}, extraParams)
-	test.That(t, err, test.ShouldBeNil)
-	test.That(t, using, test.ShouldBeFalse)
-
-	// test obstacles will cause this to error
-	_, err = ur.useURHostedKinematics(&referenceframe.WorldState{Obstacles: gifs}, nil)
-	test.That(t, err, test.ShouldNotBeNil)
-	test.That(t, err.Error(), test.ShouldResemble, errURHostedKinematics)
-}
+// TODO: Refactor this test to fix whats being tested
+// func TestUseURHostedKinematics(t *testing.T) {
```

Comment:
What this was testing is "will we get an error if we pass a worldstate while `arm_hosted_kinematics` is true". Since it's no longer possible to pass a worldstate, this can be removed entirely; there is no way to test this properly without using real hardware.

Reply:
Yeah, that was my first thought, but I commented it out to come back and confirm that was the case later. Turns out, you provided the confirmation I needed!